packages/google_maps_flutter/google_maps_flutter_web/lib/src/overlay.dart

```diff
@@ -28,6 +28,8 @@ class TileOverlayController {
     final ImageElement img =
         ownerDocument!.createElement('img') as ImageElement;
     img.width = img.height = logicalTileSize;
+    img.hidden = true;
+    img.setAttribute('decoding', 'async');
```

Comment:
Oh, cool.

Reply:
I brought back `hidden` as well, I'm not sure if it's because I was rendering different tiles, but it did seem faster to scroll the map around with it... No scientific perf measure there, just used my "feelings" haha